build/Build.pm

```diff
@@ -510,7 +510,6 @@ sub stage_distro {
     }
 
     # Un-tar the distribution.
-    my $tar;
```

Comment:
might be better to comment out?
(see L.520-521)

Reply:
https://github.com/movabletype/movabletype/pull/776/commits/bc0e43e597f1246e8221472b099ea08302688013